client/lib/i18n-utils/switch-locale.js

```diff
@@ -105,6 +107,33 @@ export async function switchCSS( elementId, cssUrl ) {
 	newLink.id = elementId;
 }
 
+function flipRTL( url, isRTL ) {
+	if ( isRTL ) {
+		return url.endsWith( '.rtl.css' ) ? url : url.replace( /\.css$/, '.rtl.css' );
+	}
+
+	return ! url.endsWith( '.rtl.css' ) ? url : url.replace( /\.rtl.css$/, '.css' );
+}
+
+function switchWebpackCSS( isRTL ) {
+	const currentLinks = document.querySelectorAll( 'link[rel="stylesheet"][data-webpack]' );
+
+	return map( currentLinks, async currentLink => {
```

Comment:
Do we need to return a map of promises? Doesn't appear it's used...

Reply:
Hmm, that's right, nobody waits for the result. That's true for the whole `switchLocale` function that does a lot of async updates -- classic CSS, webpack CSS, loading the translation file... We just call `switchLocale` and never look back.

I think it's still "good manners" to return such a promise, so I'm reluctant to remove the code. 🎵 We shall surely use it one day 🎵